### PR TITLE
[docs] Add information about key combinations on a11y sections

### PR DESCRIPTION
**Pull request by arthurbalduini**
Adds information about key combinations that might be necessary on some devices.

The design and content are open to discussion

### CODE REVIEW

File: docs/data/data-grid/accessibility/accessibility.md [hunk 1]
```diff
@@ -107,13 +107,20 @@ renderCell: (params) => (
 ### Navigation
 
 :::info
-The following key assignments apply to Windows and Linux users.
+The key assignments on the table below apply to Windows and Linux users.
```
Comment (arthurbalduini) [inline docs/data/data-grid/accessibility/accessibility.md:110]: Revised this content with the intention to clarify which 'key assignments' we are referring to, as the list of key equivalences has now grown slightly larger.

File: docs/data/data-grid/accessibility/accessibility.md [hunk 2]
```diff
@@ -107,13 +107,20 @@ renderCell: (params) => (
...
 
 - <kbd class="key">Ctrl</kbd> with <kbd class="key">⌘ Command</kbd>
 - <kbd class="key">Alt</kbd> with <kbd class="key">⌥ Option</kbd>
 
+Some devices may lack certain keys, requiring the use of key combinations. In this case, replace:
+
+- <kbd class="key">Page Up</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Up</kbd>
+- <kbd class="key">Page Down</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Down</kbd>
+- <kbd class="key">Home</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Left</kbd>
+- <kbd class="key">End</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Right</kbd>
```
Comment (LukasTy) [inline docs/data/data-grid/accessibility/accessibility.md:122]: WDYT about going with upper case letters for all the keys?
It is in lowercase only on MacBook. 🤔 🤷 
```suggestion
- <kbd class="key">Page Up</kbd> with <kbd class="key">Fn</kbd>+<kbd class="key">Arrow Up</kbd>
- <kbd class="key">Page Down</kbd> with <kbd class="key">Fn</kbd>+<kbd class="key">Arrow Down</kbd>
- <kbd class="key">Home</kbd> with <kbd class="key">Fn</kbd>+<kbd class="key">Arrow Left</kbd>
- <kbd class="key">End</kbd> with <kbd class="key">Fn</kbd>+<kbd class="key">Arrow Right</kbd>
```

File: docs/data/tree-view/accessibility/accessibility.md [hunk 3]
```diff
@@ -29,10 +29,15 @@ The [WAI-ARIA Authoring Practices](https://www.w3.org/WAI/ARIA/apg/patterns/tree
 ## Keyboard interactions
 
 :::info
-The following key assignments apply to Windows and Linux users.
+The key assignments on the table below apply to Windows and Linux users.
```
Comment (LukasTy) [inline docs/data/tree-view/accessibility/accessibility.md:32]:
```suggestion
The key assignments in the table below apply to Windows and Linux users.
```

File: docs/data/tree-view/accessibility/accessibility.md [hunk 4]
```diff
@@ -29,10 +29,15 @@ The [WAI-ARIA Authoring Practices](https://www.w3.org/WAI/ARIA/apg/patterns/tree
...
 
 On macOS replace <kbd class="key">Ctrl</kbd> with <kbd class="key">⌘ Command</kbd>.
 
+Some devices may lack certain keys, requiring the use of key combinations. In this case, replace:
+
+- <kbd class="key">Home</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Left</kbd>
+- <kbd class="key">End</kbd> with <kbd class="key">fn</kbd>+<kbd class="key">Arrow Right</kbd>
```
Comment (LukasTy) [inline docs/data/tree-view/accessibility/accessibility.md:39]:
```suggestion
- <kbd class="key">Home</kbd> with <kbd class="key">Fn</kbd>+<kbd class="key">Arrow Left</kbd>
- <kbd class="key">End</kbd> with <kbd class="key">Fn</kbd>+<kbd class="key">Arrow Right</kbd>
```

**Review by LukasTy — APPROVED** [review]
LGTM. 👌 
Thanks for a nice addition! 👍

File: docs/data/data-grid/accessibility/accessibility.md (hunk 2 above)
Comment (arthurbalduini) [reply docs/data/data-grid/accessibility/accessibility.md:122]: Agreed 👍

**Pull request merged**